provider/cmd/pulumi-resource-cloud-toolkit-aws/landingZone/accountIamArgs.ts

```diff
@@ -0,0 +1,41 @@
+export interface AccountIamArgs {
```

Comment:
I would add comments as doc for every interface that is exported and is available to the end user.